dev/com.ibm.ws.microprofile.faulttolerance_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/FATSuite.java

```diff
@@ -10,10 +10,20 @@
  *******************************************************************************/
 package com.ibm.ws.microprofile.faulttolerance.tck;
 
+import org.junit.ClassRule;
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
 import org.junit.runners.Suite.SuiteClasses;
 
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.RepeatTests;
+
 @RunWith(Suite.class)
 @SuiteClasses({ FaultToleranceTckLauncher.class })
-public class FATSuite {}
+public class FATSuite {
+
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification()
```

Comment:
Would it be better to say repeat with EE7 andWith EE8 ... then the bit below says skip for EE7

Reply:
Hmmm. I've never tried a test suite without a `withoutModification()` so I don't know how it would behave. If I had to guess it would go through server.xmls and swap cdi-1.2 for cdi-1.2

Harmless but less optimised than `withoutModification()`.